test/e2e/helpers.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const { promises: fs } = require('fs');
 const BigNumber = require('bignumber.js');
 const mockttp = require('mockttp');
+const { Browser } = require('selenium-webdriver');
```

Comment:
what is the reason for this import being needed?

Reply:
Oops, this is leftover from a change that was removed.

Reply:
Fixed here: https://github.com/MetaMask/metamask-extension/pull/20265/commits/c9cbf18fc96737926697e09dc82ea75c26829258